crates/yas_core/src/tests/test_contracts/test_yas_pool.cairo

```diff
@@ -1769,12 +1820,8 @@ mod YASPoolTests {
                 let mut amount_to_swap = IntegerTrait::<i256>::new(0, false); //Zeroable::zero();
                 if *swap_case.has_exact_out {
                     if *swap_case.exact_out { //exact OUT
-                        if *swap_case
-                            .zero_for_one { //so i check how much i should put swap IN in order to get those OUT tokens, the Asserts will still verify everything else
-                            amount_to_swap = *expected.amount_0_delta;
-                        } else {
-                            amount_to_swap = *expected.amount_1_delta;
-                        }
+                        amount_to_swap =
+                            IntegerTrait::<i256>::new(*swap_case.amount_specified.mag, true);
```

Comment:
This answer is really important because this function affects the whole test suite

Reply:
This is because of PR #194 , this is the poolCase that triggered that fix, and it had to be tested somewhere relevant. This branch won't be merged until #194 is merged first